From e42d1d265e7ebc47b5e778007fccbda02f85cbfd Mon Sep 17 00:00:00 2001
From: Johannes Segitz <jsegitz@suse.de>
Date: Thu, 16 Feb 2023 11:14:45 +0100
Subject: [PATCH 05/10] Allow systemd-timesyncd to bind to generic UDP ports
 (bsc#1207962). The watch permissions reported are already fixed in a current
 policy.

---
 policy/modules/system/systemd.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index cfa72f425..f38ff3a0e 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1235,6 +1235,7 @@ corecmd_exec_shell(systemd_timedated_t)
 corecmd_dontaudit_access_check_bin(systemd_timedated_t)
 
 corenet_tcp_connect_time_port(systemd_timedated_t)
+corenet_udp_bind_generic_node(systemd_timedated_t)
 
 dev_rw_realtime_clock(systemd_timedated_t)
 dev_write_kmsg(systemd_timedated_t)
-- 
2.55.0

